redhat-appstudio/service-provider-integration-operator
# Add timeout cleanup

### What does this PR do?

Adds configurable TTL for tokens and bindings.


### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://issues.redhat.com/browse/SVPI-65

### How to test this PR?

 - Use `mshaposh\spio:latest`
 - Configure deployment with some smaller timeouts for testing  (`--token-ttl`, `--binding-ttl` or `kubectl set env deployment/spi-controller-manager TOKENLIFETIMEDURATION=1m -n spi-system`)
 - Create objects and make sure they're deleted after timeout reached.
